civictechprojects/static/css/partials/_AboutUser.scss

```diff
@@ -1,16 +1,55 @@
+.AboutUser-root .left-column {
+  border-right: solid;
+  border-width: 1px;
+  border-color: $color-disabled-border;
```

Comment:
This should only apply in a media query for desktop; the border shows up on mobile/tablet where it's unintended:

![image](https://user-images.githubusercontent.com/16870466/123302480-98b23000-d4d1-11eb-8712-147f108ea95d.png)

(hard to see but that border right which stops partway down is from this rule)